### DISCUSSION TITLE
Dealing with javascript / jquery

### DISCUSSION THREAD

**Discussion by carlagomesah** [rubycdp/ferrum#342] [Q&A] [answered]
I'm building a script with ferrum that works fine till I submit the form and while the page is loading (I see the spinner), a new blank window opens. I'm thinking that somehow it's not dealing correctly with javascript/jquery. Wondering if anyone has any suggestions

**Comment by ttilberg** [comment, marked as answer]
Hi @carlagomesah,

Ferrum doesn't explicitly deal with anything related to javascript, it just gives directions and receives content from Chrome. Sometimes you can add rules that describes how Chrome should handle certain content, or a website may implement things to determine that you are not human and cause loading issues.

Without more information, such as the site and form in question, it's very hard to speculate what may be happening for you. As it stands, I would simply offer that Ferrum handles javascript and jquery exactly as Chrome handles it, as Chrome is in fact handling it.
